yoda/gas.go

```diff
@@ -18,8 +18,8 @@ const (
 	hasFlatGas      = 1000
 
 	// Request components
-	baseRequestSize    = uint64(32)
-	addressSize        = uint64(20)
+	baseRequestSize    = uint64(170)
+	addressSize        = uint64(52)
```

Comment:
Can you explain why use 52 for address size?

Reply:
I checked on `RequestedValidators` in `Request` Message. When I increase 1 validator, the byte length of the request is increased by 52.